src/pages/CreateEventPage/CreateEventPage.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from 'react';
+import { CreateEventWrapper, Form } from './styled';
+import { formatISO, format } from 'date-fns';
+import { createEvent } from '../../api/events';
+import TextField from '../../components/TextField';
+import DateField from '../../components/DateField';
+import TimeField from '../../components/TimeField';
+import SelectField from '../../components/SelectField';
+import TextAreaField from '../../components/TextAreaField';
+import Alert from '../../components/Alert';
+
+function CreateEvent() {
+  const [name, setName] = useState('');
+  const [image, setImage] = useState('');
+  const [imageAlt, setImageAlt] = useState('');
+  const [date, setDate] = useState(format(Date.now(), 'yyyy-MM-dd'));
+  const [starttime, setStarttime] = useState(format(Date.now(), 'HH:mm'));
+  const [endtime, setEndtime] = useState(format(Date.now(), 'HH:mm'));
+  const [description, setDescription] = useState('');
+  const [eventType, setEventType] = useState('');
```

Comment:
Alternatively, you could initialize it as `null` with the following expression `useState<null | string>(null)`.

On another note, we've yet to decided whether this property should be included, but I do see the merit.